package.json

```
@@ -289,6 +290,68 @@
           "default": true,
           "description": "Enable cursor blinking in terminal."
         },
+        "secondaryTerminal.fontSize": {
```

Biome (2.1.2)

[error] The key secondaryTerminal.fontSize was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)